src/main/java/org/takes/facets/auth/PsAll.java

```diff
@@ -60,12 +60,13 @@ public PsAll(final List<? extends Pass> passes, final int identity) {
     }
 
     @Override
-    public final Opt<Identity> enter(final Request request) throws IOException {
-        final Opt<Identity> result;
+    public final Optional<Identity> enter(final Request request)
+        throws IOException {
+        final Optional<Identity> result;
         if (this.allMatch(request)) {
             result = this.all.get(this.index).enter(request);
         } else {
-            result = new Opt.Empty<Identity>();
+            result = new Optional<>(null);
```

Comment:
@Happy-Neko why change a perfectly fine `Opt.Empty` to a optional that is called with a `null`? This is ugly.